cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,5 +1,5 @@
 import {MenuContentPage, AddressStepPage, LoginPage, PaymentStepPage,
-  ProductsListPage, ShippingStepPage, ShoppingCartPage} from "../page/index";
+  ProductsListPage, ShippingStepPage, ShoppingCartPage, GeneralStepPage} from "../page/index";
 
 const menuContentPage = new MenuContentPage();
 const addressStepPage = new AddressStepPage();
```

Comment:
I'm seeing a warning that this instance is not being used anywhere, please check

Reply:
Done